pkg/api/v1/router_instance_metadata.go

```diff
@@ -70,6 +70,14 @@ func (r *Router) instanceMetadataGet(c *gin.Context) {
 		return
 	}
 
+	// Treat metadata that's older then its TTL as non-existent
+	if metadata != nil && viper.GetDuration("cache_ttl") != 0 && metadata.UpdatedAt.Add(viper.GetDuration("cache_ttl")).Before(time.Now()) {
```

Comment:
perhaps the ttl should be cached on the router so we're not having to re parse it for every request.

Reply:
I tried this, and I found that I could no longer change the value at runtime during the tests, even with some plumbing in the test mocks for the http server/router.